Library/Formula/unifdef.rb

```diff
@@ -13,6 +13,6 @@ def install
   end
 
   test do
-    system "echo '' | #{bin}/unifdef"
+    pipe_output("echo '' | #{bin}/unifdef")
```

Comment:
If you're using `pipe_output` instead of using `|` to pipe you can pass it as (I believe) the second argument.